tests/src/JIT/Directed/VectorABI/VectorMgdMgdArray.cs

```diff
@@ -0,0 +1,1099 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.Intrinsics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// Test passing and returning HVAs (homogeneous vector aggregates) to/from managed code.
+// Test various sizes (including ones that exceed the limit for being treated as HVAs),
+// as well as passing various numbers of them (so that we exceed the available registers),
+// and mixing the HVA parameters with non-HVA parameters.
+
+// This Test case covers all cases for
+//   Methods that take one HVA argument with between 1 and 9 Vectror64 or Vector128 elements
```

Comment:
typo: (Vectror64 => Vector64) - this might be originally my typo ;-)
Also, here and below you should change the '9' to '5' since that's what's actually being tested.